src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.css

```diff
@@ -6,3 +6,15 @@
 .tagWrapper > span {
   margin-left: 0;
 }
+
+.text {
+  margin: 0 6px;
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  color: var(--temp-grey-blue-7);
+}
+
+.progressBarContainer {
+  margin-left: 4px;
+  width: 116px;
+}
```

Comment:
I think the progress bar should be fixed, otherwise it jumps around based on the number of seconds.

See this example:
![bug-vr-voting-treshold](https://user-images.githubusercontent.com/1193222/116408603-aa61b900-a83b-11eb-9b09-3bb5efff7cfe.gif)